tests/e2e/addons/TA_cim_addon/samples/test_sample

```diff
@@ -0,0 +1 @@
+{"actor": {"id": "spr6w8a2l2V88E3nk5d7", "type": "SystemPrincipal", "alternateId": "system@okta.com", "displayName": "Okta System", "detailEntry": null}, "client": {"userAgent": {"rawUserAgent": "Splunk Add-on for Okta Identity Cloud", "os": "Unknown", "browser": "UNKNOWN"}, "zone": "null", "device": "Unknown", "id": null, "ipAddress": "##ip_address##",  "destadd": "0:0:0:0:0:0:0:1", "geographicalContext": null}, "device": null, "authenticationContext": {"authenticationProvider": null, "credentialProvider": null, "credentialType": null, "issuer": null, "interface": null, "authenticationStep": 0, "externalSessionId": null}, "displayMessage": "Blocked request from IP: 68c6:7554::89", "eventType": "security.request.blocked", "outcome": {"result": "SUCCESS", "reason": "NETWORK_ZONE_BLACKLIST"}, "published": "2023-10-11T10:28:27.114Z", "securityContext": {"asNumber": null, "asOrg": null, "isp": null, "domain": null, "isProxy": null}, "severity": "WARN", "debugContext": {"debugData": {"requestId": "ZSZ4y03BLKYbIGjkjkgRvgAADhk", "requestUri": "/api/v1/logs", "url": "/api/v1/logs?limit=1000&since=2023-10-11T09%3A01%3A17.417Z&until=2023-10-11T10%3A27%3A56.967Z"}}, "legacyEventType": "security.zone.request.blocked", "transaction": {"type": "WEB", "id": "ZSZ4y03BLKYbIGjkjkgRvgAADhk", "detail": {}}, "uuid": "ea1dfcef-6820-11ee-bb11-f3c3bf8f14d1", "version": "0", "request": {"ipChain": [{"ip": "68c6:7554::89", "geographicalContext": null, "version": "V4", "source": null}]}, "target": null}]]
```

Comment:
Should `destadd` be hardcoded?

Reply:
It does not matter because we are tokenizing ip_Addresee and that is extracted as src_ip, so destadd does not matter really.

Reply:
So then we don't need `FIELDALIAS-client_destadd_as_dest_ip = client.destadd AS dest_ip` line in `props.conf`?

Reply:
yes